frontend/ui/screensaver.lua

```diff
@@ -375,10 +381,33 @@ function Screensaver:show(event, fallback_message)
             screensaver_message = self:expandSpecial(screensaver_message, fallback)
         end
 
-        widget = InfoMessage:new{
-            text = screensaver_message,
-            readonly = true,
-        }
+        if messagePos == "middle" or messagePos == nil then
+            widget = InfoMessage:new{
+                text = screensaver_message,
+                readonly = true,
+            }
+        else
+            local face = Font:getFace("infofont")
+            local container
+            if messagePos == "bottom" then container = BottomContainer
+            else container = TopContainer
```

Comment:
I find that clearer:
```lua
            if messagePos == "bottom" then
               container = BottomContainer
            else
               container = TopContainer
            end
```
or just
```lua
               container = messagePos == "bottom" and BottomContainer or TopContainer
```

Reply:
Sleep-deprived brain says +1 for the first variant ;).

Reply:
Also don't forget this comment.

Reply:
Also don't forget this comment.